modules/nf-core/cellranger/mkvdjref/main.nf

```diff
@@ -0,0 +1,39 @@
+process CELLRANGER_MKVDJREF {
+    tag "$fasta"
+    label 'process_high'
+
+    container "nfcore/cellranger:7.1.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "CELLRANGER_MKREF module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    path fasta
```

Comment:
Is there a possibiliy a pipeline may allow multiple references? If so I would suggest adding `meta` here (and you can then change the `tag` to meta.id)

Reply:
> the generation of multiple references from separate organisms would simply make separate calls to cellranger mkvdjref

Yes but tha'ts exactly my point. How would you track which output comes from which call?

I'll leave it for now as I know you've been working on this for a while - but a reviewer may bring it up again in the future.